src/commands/createNewProject/createNewProject.ts

```diff
@@ -54,6 +56,13 @@ export async function createNewProjectInternal(context: IActionContext, options:
     const wizardContext: Partial<IFunctionWizardContext> & IActionContext = Object.assign(context, options, { language, version: tryParseFuncVersion(version), projectTemplateKey });
     const optionalExecuteStep = options.executeStep;
 
+    if (optionalExecuteStep instanceof CreateDockerfileProjectStep) {
+        const message: string = localize('installFuncTools', 'You must have the Azure Functions Core Tools installed to run this command.');
```

Comment:
nit: Since we know what command they are running, I think we should tell them specifically that they need to have the core tools installed to create a containerized function project or whatever.